test/core/integration/concrete/lockup-dynamic/get-segments/getSegments.t.sol

```diff
@@ -1,17 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.22 <0.9.0;
 
 import { Errors } from "src/core/libraries/Errors.sol";
-import { LockupDynamic } from "src/core/types/DataTypes.sol";
+import { Lockup, LockupDynamic } from "src/core/types/DataTypes.sol";
 
-import { Lockup_Dynamic_Integration_Shared_Test } from "./../LockupDynamic.t.sol";
+import { Integration_Test } from "./../../../Integration.t.sol";
 
-contract GetSegments_Integration_Concrete_Test is Lockup_Dynamic_Integration_Shared_Test {
+contract GetSegments_Integration_Concrete_Test is Integration_Test {
```

Comment:
For symmetry, I would like to see it inherited from `Lockup_Dynamic_Integration_Shared_Test`. The reason is simple, symmetry makes it easy to change default model in base contract otherwise changing default model in base contract would mean refactor which should not be the case.